wix-ecom-backend/wix-ecom-backend/OrdersSettings/guides/Introduction.md

```diff
@@ -37,9 +37,10 @@ import { ordersSettings } from 'wix-ecom-backend';
 
 ## Permissions information
 
-The following functions in the Orders Settings API are restricted and only run if you elevate permissions using the `wix-auth` [`elevate()`](https://www.wix.com/velo/reference/wix-auth/elevate) function:
+The following functions in the Orders Settings API are restricted and only run if
+you elevate permissions using the `wix-auth` [`elevate()`](https://dev.wix.com/docs/velo/api-reference/wix-auth/elevate) function:
```

Comment:
Why the new line?

Reply:
just for IDE reading convenience.
it will render as the same line